gtsam/linear/GaussianBayesNet.h

```diff
@@ -82,6 +82,12 @@ namespace gtsam {
     /** Check equality */
     bool equals(const This& bn, double tol = 1e-9) const;
 
+    /// Check exact equality.
+    friend bool operator==(const GaussianBayesNet& lhs,
+                           const GaussianBayesNet& rhs) {
+      return lhs.isEqual(rhs);
```

Comment:
But this is not exact equality

Reply:
The docstring for the `isEqual` method in `FactorGraph.h` says
```
/// Check exact equality of the factor pointers. Useful for derived ==.
```
You wrote that docstring 3 years ago.

Reply:
Ah, isEqual might be. Please check it. `equals`is not.